src/python/WMCore/Services/DBS/DBS3Reader.py

```diff
@@ -592,8 +597,8 @@ def listFileBlockLocation(self, fileBlockName, dbsOnly = False, phedexNodes=Fals
                 msg += "%s\n" % formatEx3(ex)
                 raise DBSReaderError(msg)
 
-            if not any(blockInfo.values()): # no data location from dbs
-                return list()
+        if not any(blockInfo.values()): # no data location
```

Comment:
Actually, the `any()` is also not needed, right? I can just do `if not blockInfo:`.